proto/sysinfo.proto

```diff
@@ -479,3 +479,27 @@ message HardwareInfo {
     }];
 
 }
+
+// OSX Plist Start Calendar Interval
+
+message LaunchdStartCalendarInterval {
+  optional int32 minute = 1;
+  optional int32 hour = 2;
+  optional int32 day = 3;
+  optional int32 weekday = 4;
+  optional int32 month = 5;
+}
+
+// OSX Plist Files
+message LaunchdPlist {
+  optional string label = 1;
+  optional bool disabled = 2;
+  optional string username = 3;
```

Comment:
Usually we would use the _ and lower case style for proto fields, but I think in this case its clearer to preserve whatever name its supposed to be from the plist. This way we can correlate which each field is supposed to be exactly. 

We do similar stuff for registry or tsk enums for example.